content/guides/use-case/nlp/language-translation.md

```diff
@@ -0,0 +1,296 @@
+---
+title: Build a language translation app
+keywords: nlp, natural language processing, text summarization, python, language translation, googletrans
+description: Learn how to build and run a language translation application using Python, Googletrans, and Docker.
+---
+
+## Overview
+
+This guide walks you through building and running a language translation
+application. You'll build the application using Python with Googletrans, and
+then set up the environment and run the application using Docker.
+
+The application demonstrates a simple but practical use of the Googletrans
+library for language translation, showcasing basic Python and Docker concepts.
+Googletrans is a free and unlimited Python library that implements the Google
+Translate API. It uses the Google Translate Ajax API to make calls to such
+methods as detect and translate.
+
+## Prerequisites
+
+* You have installed the latest version of [Docker Desktop](../../../get-docker.md). Docker adds new features regularly and some parts of this guide may work only with the latest version of Docker Desktop.
+* You have a [Git client](https://git-scm.com/downloads). The examples in this section use a command-line based Git client, but you can use any client.
+
+## Get the sample application
+
+1. Open a terminal, and clone the sample application's repository using the
+   following command.
+   ```console
+   $ git clone https://github.com/harsh4870/Docker-NLP.git
+   ```
+2. Verify that you cloned the repository.
+
+   You should see the following files in your `Docker-NLP` directory.
+   ```text
+   01_sentiment_analysis.py
+   02_name_entity_recognition.py
+   03_text_classification.py
+   04_text_summarization.py
+   05_language_translation.py
+   entrypoint.sh
+   requirements.txt
+   Dockerfile
+   README.md
+   ```
+
+## Explore the application code
+
+The source code for the application is in the
+`Docker-NLP/05_language_translation.py` file. Open `05_language_translation.py`
+in a text or code editor to explore its contents in the following steps.
+
+1. Import the required libraries.
+   ```python
+   from googletrans import Translator
+   ```
```

Comment:
while not required by commonmark, I wonder if maybe we should consistently add blank lines before and after code blocks. I find that it improves readability and, aside from aesthetics, some parser would have problems with the lack of blank lines.

The same is true for lists. (A list can be tight, but blank lines before and after is usually a good idea, and makes it easier to catch unwanted [lazy continuation lines](https://spec.commonmark.org/0.31.2/#lazy-continuation-line)).